### ISSUE TITLE
plot_method="plotly" - produces a white space between dendrograms and matrix

### ISSUE THREAD

**Issue by talgalili** [talgalili/heatmaply#47]
Example:

```r
library(heatmaply)
heatmaply(mtcars, 
          plot_method="plotly")

```

![image](https://cloud.githubusercontent.com/assets/976006/24837620/d49a5cb6-1d40-11e7-8e64-3593a35f42e0.png)

**Comment by Alanocallaghan** [comment]
I noticed this but haven't gotten to the bottom of it yet, it's quite annoying. It doesn't seem to  be related to plot padding

**Comment by Alanocallaghan** [comment]
Partial fix; issue apparently relates to axis range on dendrograms

**Comment by Alanocallaghan** [comment]
Incidentally a similar bug currently exits when `row_dend_left=TRUE` and `plot_method=ggplot`. In this case half of the topmost and bottom-most cells are cut off due to the limits being set by the row dendrogram. This should be fixed by #54

**Comment by talgalili** [comment]
Great to have this solved. Thanks.
